pkgs/applications/editors/vim/plugins/get-plugins.nix

```
@@ -1,19 +1,28 @@
-with import <localpkgs> {};
+with import <localpkgs> { };
 let
-  inherit (vimUtils.override {inherit vim;}) buildVimPlugin;
+  inherit (vimUtils.override { inherit vim; }) buildVimPlugin;
   inherit (neovimUtils) buildNeovimPlugin;
 
   generated = callPackage <localpkgs/pkgs/applications/editors/vim/plugins/generated.nix> {
     inherit buildNeovimPlugin buildVimPlugin;
-  } {} {};
-  hasChecksum = value:
-    lib.isAttrs value && lib.hasAttrByPath ["src" "outputHash"] value;
-  getChecksum = name: value:
-    if hasChecksum value then {
-      submodules = value.src.fetchSubmodules or false;
-      sha256 = value.src.outputHash;
-      rev = value.src.rev;
-    } else null;
+  } { } { };
+  hasChecksum =
+    value:
+    lib.isAttrs value
+    && lib.hasAttrByPath [
+      "src"
+      "outputHash"
+    ] value;
```

As a nixpkgs-fmt user this feels ugly.
I wonder if this would be ok? I think it's cleaner to use the built-in syntax anyway.

```suggestion
    && (value?src.outputHash);
```

---

Indeed this can be simplified and reformatted to just
```
  hasChecksum = value: value ? src.outputHash;
```

[Relevant motto](https://github.com/NixOS/rfcs/blob/62d1245ec1eae275edb996229585637035c02538/rfcs/0166-nix-formatting.md?plain=1#L78) from the RFC:

> *Bad code does not deserve good formatting.*

:laughing: